python/lsst/pipe/supertask/config.py

```diff
@@ -47,6 +49,66 @@
 # ------------------------
 
 
+def makeDatasetField(name, dtype, removeUnitsVariable=False):
```

Comment:
This function needs a docstring. If it's an internal function (it's not in `__all__`) then consider using a leading underscore in the name.